.env.sample

```diff
@@ -1,10 +1,15 @@
-GOV_NOTIFY_API_KEY=
-EMAIL_APPLICATION_RECEIVED_TEMPLATE_ID=
-HACKNERY_AUTH_URL=
-POSTCODE_LOOKUP_URL=
-POSTCODE_LOOKUP_APIKEY=
-URL_PREFIX=
+#This file outlines the environment variables this application needs
+#note that anything not commented will be loaded by the application
+
+#GOV_NOTIFY_API_KEY=
+#EMAIL_APPLICATION_RECEIVED_TEMPLATE_ID=
+#HACKNERY_AUTH_URL=
```

Comment:
This is a typo that we may or may not want to fix at some point.

Reply:
I'd like to do that I'm just not sure how far the typo's propagated